frontend/src/_layouts/base.html

```diff
@@ -1,16 +1,16 @@
 <!DOCTYPE html>
-<html lang="en" class="font-sans text-gray-600">
+<html class="font-sans text-gray-600" lang="en">
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
     <meta name="theme-color" content="#115E59" />
     <meta name="msapplication-navbutton-color" content="#115E59 " />
-    <link rel="manifest" href="./manifest.webmanifest" />
-    <link rel="icon" sizes="32x32" type="image/png" href="images/favicon-32x32.png" />
-    <link rel="icon" sizes="16x16" type="image/png" href="images/favicon-16x16.png" />
-    <link rel="stylesheet" href="./app.css" />
+    <link rel="manifest" href="~/src/manifest.webmanifest" />
+    <link rel="icon" sizes="32x32" type="image/png" href="~/src/images/favicon-32x32.png" />
+    <link rel="icon" sizes="16x16" type="image/png" href="~/src/images/favicon-16x16.png" />
+    <link rel="stylesheet" href="~/src/app.css" />
 
-    <title>Srila BV Narayana Gosvami Maharaja’s Official Archive</title>
+    <title>{{ title }}</title>
```

Comment:
I think this common title "Srila BV Narayana Gosvami Maharaja’s Official Archive" should go to the global data, and here we can concatenate particular page title (such as "Audios") with the generic title, like we see on many websites. See the best practices for this in 11ty.

Reply:
changed.